### PR TITLE
LPS-187554 [PRM] Display status bars at Renewals Dashboard

### PR DESCRIPTION
**Pull request by luisacmn**

### CODE REVIEW

**Comment by luisacmn** [comment]
ci:test:sf

**Comment by luisacmn** [comment]
ci:test:acceptance

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/dashboard/RenewalsChart/index.tsx [hunk 1]
```diff
@@ -102,6 +102,7 @@ export default function () {
 									'mr-3 status-bar-vertical',
 									getCurrentStatusColor(item)
 								)}
+								style={{height: '40px', width: '10px'}}
```
Comment (JoMaAlves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/dashboard/RenewalsChart/index.tsx:105]: Could you add this to a css file, just like the other charts?

**Review by JoMaAlves — CHANGES_REQUESTED** [review]
🛠️ Needs some changes 🛠️

**Review by JoMaAlves — APPROVED** [review]
LGTM ​🦖​🌋​ 
Thank you @luisacmn

**Comment by JoMaAlves** [comment]
ci:forward

**Comment by Manda-vit** [comment]
resent [#136924](https://github.com/brianchandotcom/liferay-portal/pull/136924)

**Pull request closed without merging**